shm_wrapper/shm_wrapper.h

```diff
@@ -310,7 +325,8 @@ int gamepad_write(uint32_t pressed_buttons, float joystick_vals[4]);
  *    value: value of the parameter, with the corresponding type filled with data
  * Returns:
  *    0 on success
- *    -1 on failure (maximum number of custom parameters is reached so this parameter isn't written)
+ *   -1 when maximum number of custom parameters is reached
+ *   -2 when the given key is longer than 63 characters
```

Comment:
Can we expect that 63 will be "hardcoded"? I can see that 63 is pretty sizable so it probably won't get any larger.

Reply:
JK this looks like this is `MAX_LOG_KEY_LENGTH - 1`? If so then we should reference that `#define` instead of mentioning 63, which may change

Reply:
yea I just forgot to update the doc